internal/tiltfile/value/string.go

```diff
@@ -17,31 +21,31 @@ func AsString(x starlark.Value) (string, bool) {
 
 // Unpack an argument that can either be expressed as
 // a string or as a list of strings.
-func AsStringOrStringList(x starlark.Value) ([]string, bool) {
+func AsStringOrStringList(x starlark.Value) ([]string, error) {
```

Comment:
bonus points if you convert this to use the Unpack() interface, so that more of the error messaging will be handled by starlark internally

Reply:
oy, of course!